ydb/core/persqueue/pq_impl.cpp

```diff
@@ -1176,22 +1283,22 @@ void TPersQueue::AggregateAndSendLabeledCountersFor(const TString& group, const
 
 void TPersQueue::Handle(TEvPQ::TEvPartitionLabeledCounters::TPtr& ev, const TActorContext& ctx)
 {
-    auto it = Partitions.find(ev->Get()->Partition);
-    Y_ABORT_UNLESS(it != Partitions.end());
+    const auto partitionId = ev->Get()->Partition;
+    auto& partition = GetPartitionInfo(partitionId);
     const TString& group = ev->Get()->LabeledCounters.GetGroup();
```

Comment:
Do not public labeled counters for shadow partitions. return right now.

Reply:
done
https://github.com/ydb-platform/ydb/pull/1037/commits/5090d78400d2f4c82c3f358033a1b8b725aea96b